animal-sniffer/src/main/java/org/codehaus/mojo/animal_sniffer/ClassFileVisitor.java

```diff
@@ -124,8 +124,7 @@ else if ( ( file.getName().endsWith( ".jar" ) || file.getName().endsWith( ".jmod
     /**
      * Recursively finds class files and invokes {@link #process(String, InputStream)}
      *
-     * @param path Directory (or other Path like {@code Paths.get(URI.create("jrt:/modules"))}) full of class files,
-     *             or a class file (in which case that single class is processed).
+     * @param path Directory (or other Path like {@code Paths.get(URI.create("jrt:/modules"))}) full of class files
```

Comment:
Changed this because this method does not actually support a `Path` which directly points to a file because processing is done in `postVisitDirectory` below, but that would never be called in that case.